target/linux/ath79/dts/ar9344_openmesh_mr600.dtsi

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9344.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	chosen {
+		/delete-property/ bootargs;
+	};
+
+	aliases {
+		serial0 = &uart;
+		label-mac-device = &eth0;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&ref {
+	clock-frequency = <40000000>;
+};
+
+&uart {
+	status = "okay";
+};
+
+&spi {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <40000000>;
+
+		/* partitions are passed via bootloader */
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "u-boot";
+				reg = <0x000000 0x040000>;
+				read-only;
+			};
+
+			partition@40000 {
+				label = "u-boot-env";
+				reg = <0x040000 0x010000>;
+			};
+
+			partition@50000 {
+				label = "custom";
+				reg = <0x050000 0x060000>;
+				read-only;
+			};
+
+			partition@b0000 {
+				label = "inactive";
+				reg = <0x0b0000 0x7a0000>;
+			};
+
+			partition@850000 {
+				label = "inactive2";
+				reg = <0x850000 0x7a0000>;
+			};
+
+			art: partition@ff0000 {
+				label = "ART";
+				reg = <0xff0000 0x010000>;
+				read-only;
+			};
+		};
+	};
+};
+
+&mdio0 {
+	status = "okay";
+
+	phy-mask = <0x1>;
+
+	phy0: ethernet-phy@0 {
+		reg = <0>;
+	};
+};
+
+&eth0 {
+	status = "okay";
+
+	pll-data = <0x06000000 0x00000101 0x00001313>;
+
+	mtd-mac-address = <&art 0x0>;
+
+	phy-mode = "rgmii";
+	phy-handle = <&phy0>;
```

Comment:
see if this works?

```
	pll-data = <0x02000000 0x00000101 0x00001313>;

	mtd-mac-address = <&art 0x0>;

	phy-mode = "rgmii-id";
	phy-handle = <&phy0>;
```

Reply:
Why should I change the pll-data from the original ar711xx mach file? I already have enough problems with the PRs because I've switched the WAN port to the gmac0 (and the mac addresses in this process) for some devices.

Reply:
Because there have been vast improvements in the at803x driver since linux 5.1

there is many cases where it doesn't matter what was in ar71xx, it is outdated as a whole

Reply:
you can read what I have discovered here

https://github.com/openwrt/openwrt/pull/3505#issuecomment-716050292

Reply:
Ok, then I will change the values in a separate commit but not in the one which tries to re-add the devices.

Btw. thanks for the link. I will try to reference it

Reply:
adding a device to ath79 is considered a completely new addition for the most part. there is no value in "preserving" the exact configuration from ar71xx, and then changing it in later commits.

not for me to decide though, you're welcome to ask the developers, but I feel like this is what they would say

if you do run into trouble you can forget about it, since it is not critical, but it should work fine...

Reply:
based on the responses I got from @adschm, it seems like the review is actually involving a comparison with the ar71xx sources. And I've already broken out some newer changes after some discussion in the OM2P PR.

And to be honest, it is rather easy to squash two commits - unsquashing is a lot harder.

Reply:
comparison yes, but like Adrian said, as a point of reference, not to make them identical 👍🏽